contributors/devel/sig-testing/writing-good-e2e-tests.md

```diff
@@ -229,3 +229,16 @@ Unreachable nodes are evacuated and then repopulated upon rejoining [Disruptive]
 Note that opening issues for specific better tooling is welcome, and
 code implementing that tooling is even more welcome :-).
 
+### Resource usage ###
+When writing tests, resources used in the tests should be chosen specifically and sanely quanified.
+
+Therefore it is important to use resources that are:
+1. appropriate to the test
+2. inexpensive (containing little or no overhead)
+3. create just as many an needed, no more
+
```

Comment:
should be cleaned up at the end of the run.

Reply:
Yes! This is an essential point.
However, isn't this done when the namespace is teared down at the end of each test?

Reply:
Not always, sometimes namespace deletes timeout because of hung finalizers.

Reply:
> Not always, sometimes namespace deletes timeout because of hung finalizers.

OK, I'll added in 2aa77f1a